docs/content/preview/benchmark/key-value-workload-ycql.md

```diff
@@ -67,7 +67,7 @@ We will use the [YugabyteDB Workload Generator](https://github.com/yugabyte/yb-s
 To get the tool (``yb-sample-apps.jar`), run the following command.
 
 ```sh
-$ wget https://github.com/yugabyte/yb-sample-apps/releases/download/1.3.9/yb-sample-apps.jar?raw=true -O yb-sample-apps.jar
+$ wget 'https://github.com/yugabyte/yb-sample-apps/releases/download/1.3.9/yb-sample-apps.jar?raw=true' -O yb-sample-apps.jar
```

Comment:
Prompt should be deleted here  (we don't include prompts because when the user copies the command and pastes it on their terminal, they end up with two prompts)